docs/1_setup.md

```diff
@@ -18,40 +18,60 @@ of developers.
+  to fetch tweets directed at the logged in user
+- The v2
+  [blocks](https://developer.twitter.com/en/docs/twitter-api/users/blocks/introduction)
   endpoint to block users on behalf of the authenticated user
-- The v2 [mutes](https://developer.twitter.com/en/docs/twitter-api/users/mutes/introduction)
+- The v2
+  [mutes](https://developer.twitter.com/en/docs/twitter-api/users/mutes/introduction)
   endpoint to mute users on behalf of the authenticated user
 - The v2 [hide
   replies](https://developer.twitter.com/en/docs/twitter-api/tweets/hide-replies/introduction)
   endpoint to hide replies on behalf of the authenticated user
 
 To support all this functionality, you'll need to [get access to the Twitter
-API](https://developer.twitter.com/en/docs/twitter-api/getting-started/getting-access-to-the-twitter-api)
-and the Enterprise Full-Archive Search API.
+API](https://developer.twitter.com/en/docs/twitter-api/getting-started/getting-access-to-the-twitter-api).
 
-Once granted, take note of the:
+**NOTE: The full suite of Twitter APIs the app uses require additional access
+beyond the default Twitter API [Essential access
+level](https://developer.twitter.com/en/docs/twitter-api/getting-started/about-twitter-api).
+The [Enterprise
+search](https://developer.twitter.com/en/docs/twitter-api/enterprise/search-api/overview)
+API requires an [enterprise
+account](https://developer.twitter.com/en/docs/twitter-api/enterprise/search-api/overview),
+while v2 Full-Archive Search requires [academic
+access](https://developer.twitter.com/en/products/twitter-api/academic-research).**
 
-- Account name, app key, and app secret for your Twitter API developer account
-- Username and password for your Enterpise Full-Archive Search API account
+Once granted access, take note of the:
 
-You'll need both sets of credentials later on.
+- Account name, app key, and app secret for your Twitter API developer account
+- If using the Enterprise Full-Archive Search API, the Username and password for
+  your enterprise account
+- If using the v2 Full-Archive Search endpoint, the Twitter bearer token for
+  your app
+
+You'll need these credentials later on.
+
+### Enterprise Full-Archive Search vs. v2 Full-Archive Search
+
+The tool is implemented in a way that either API can be used. While both APIs
+offer similar functionality, there are key differences in rate limits. We refer
+users to [Twitter's
+comparison](https://developer.twitter.com/en/docs/twitter-api/tweets/search/migrate)
+for more details. You may also see minute differences in:
+
+- Which tweets are fetched. This is due to differences in granularity for
+  timestamp format each API supports (YYYYMMDD for Enterprise and
+  YYYY-MM-DDTHH:mm:ssZ for v2).
+- The order the tweets are displayed when sorted by "Priority". This is due to
+  small differences in how the APIs return the tweet text, which causes some
```

Comment:
Could you clarify what you mean here by small differences? (No documentation update required, I'm just curious).

Reply:
I dug into the difference a bit and turns out it's not actually that the APIs return different text. It seems more like the differences are due to tiny bugs in how we're stripping out URLs, hashtags, etc. in Enterprise. I created issue #19 to fix that and updated the text here accordingly.